README.md

```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
+
+In your Unity project with Arcadia, do:
+
+1. Create a new "regular" looking clojure project that has a deps.edn at its root. For example:
+
+  ```sh
+cd Assets
+mkdir <project-name>
+cd <project-name>
+mkdir src test target
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+              arcadia             {:local/root "../Arcadia"}}}' > deps.edn
+```
+
+2. Open and import the project with Cursive at that root.
+3. You should now have auto complete with the arcadia and clojure-clr sources.
+
+Now we'll want to talk to Arcadia via nREPL. To do this:
+
+1. Clone this repository: git@github.com:eponai/tools.cursive-arcadia-repl.git
```

Comment:
The URL is now rendered properly, only the first part. Perhaps explicitly link this instead?

Reply:
https://github.com/eponai/tools.cursive-arcadia-repl/pull/6/files#r285394099